test/generators/java/presets/__snapshots__/ConstraintsPreset.spec.ts.snap

```diff
@@ -10,7 +10,7 @@ exports[`JAVA_CONSTRAINTS_PRESET should render constraints annotations 1`] = `
   private double maxNumberProp;
   @Size(min=2, max=3)
   private Object[] arrayProp;
-  @Pattern(regexp=\\"^I_\\")
+  @Pattern(regexp=\\"^\\\\\\\\w+(\\\\\\"\\\\\\\\.\\\\\\\\w+)*$\\")
```

Comment:
Just double checking this looks right to you @dlkj 😄

Reply:
this does look right, but let's step through it slowly.

The diff is part of an escaped  JS string capturing the output of the test. This has been escaped twice.

`"@Pattern(regexp=\\"^\\\\\\\\w+(\\\\\\"\\\\\\\\.\\\\\\\\w+)*$\\")"`

-->

`@Pattern(regexp=\"^\\\\w+(\\\"\\\\.\\\\w+)*$\")`

-->

`@Pattern(regexp="^\\w+(\"\\.\\w+)*$")`